obspy/signal/headers.py

```diff
@@ -147,6 +147,14 @@
     C.c_int]
 clibsignal.calculate_kernel.restype = None
 
+clibsignal.aic_simple.argtypes = [
+    np.ctypeslib.ndpointer(dtype=np.float64, ndim=1,
```

Comment:
@QuLogic is the np.float64 needed here or it's also part of the alias-ing you solved recently?

Reply:
same L153

Reply:
Yes, they are needed here. You need to tell a shared library what's the data type for input and output. `np.ctypeslib` provides a handy tool to convert a numpy dtype to a correct C array type.

Reply:
my meaning is that ``np.float64`` is an alias to builtin ``float``, which is something @QuLogic spent time cleaning in our codebase.

Reply:
I see. Well, I did what other functions do in the `headers.py` file.

Reply:
I think @QuLogic spent time getting rid of the `np.float` usages, which are just an alias to the python `float`. `np.float64` should be ok.

Reply:
I prefer `np.float64` over plain `float` because it's actually double and might be confused with C-float plain data type.

Reply:
`np.float64` is fine; it's `np.float` that is deprecated.